modules/python/test/test_cuda.py

```diff
@@ -33,6 +33,8 @@ def test_cuda_interop(self):
         self.assertTrue(cuMat.cudaPtr() != 0)
         stream = cv.cuda_Stream()
         self.assertTrue(stream.cudaPtr() != 0)
+        asyncstream = cv.cuda_Stream(1)
```

Comment:
Consider adding a comment with description for magic numbers:

```
asyncstream = cv.cuda_Stream(1)  # cudaStreamNonBlocking
```

Reply:
The CUDA api defines are documented in the doxygen for the constructor. I can also add it to this test file, no problem.